internals-js/src/error.ts

```diff
@@ -561,6 +561,12 @@ const INTERFACE_KEY_MISSING_IMPLEMENTATION_TYPE = makeCodeDefinition(
   { addedIn: '2.3.0' },
 )
 
+const SOURCE_FEDERATION_VERSION_REQUIRED = makeCodeDefinition(
+  'SOURCE_FEDERATION_VERSION_REQUIRED',
+  'Schemas using `@source{API,Type,Field}` directives must @link-import v2.7 or later of federation',
```

Comment:
is it more accurate/useful to say that `schemas linking to "https://specs.apollo.dev/source" must link to federation version 2.7 or later`?

Reply:
https://github.com/apollographql/federation/pull/2914/commits/7ced7638606529424950b47d98fbb3338066ff50